wpeverest/everest-forms
# Fixes - WordPress 5.3 UI compatibility bug

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes overall broken UI after the update of WordPress version 5.3.

### How to test the changes in this Pull Request:

1. Update the WordPress version to 5.3.
2. Check the overall design.

### Types of changes:




### Other information:




### Changelog entry

> Fixes - UI fixes after update of WP v5.3
